### ISSUE TITLE
Heading on Dashboards and Downloads pages should be h1

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/rialto-web#137]
Currently the heading for each dashboard page ("Publications Dashboard", "Open Access Dashboard", "ORCID iD Adoption Dashboard") and the "Downloads" page, is an `h2`. We should change these to `h1`.
